cypress/integration/app_spec.ts

```diff
@@ -22,9 +22,21 @@ function setupServerMocks() {
     "https://freesound.org/data/previews/462/462808_8386274-lq.mp3",
     { fixture: "sound.mp3" }
   );
-  cy.intercept("GET", "https://freesound.org/apiv2/search/text/*", {
-    fixture: "search-results.json",
-  });
+  cy.intercept(
+    "GET",
+    "https://freesound.org/apiv2/search/text/?fields=id%2Cname%2Cpreviews%2Cduration%2Cnum_downloads%2Cusername%2Cnum_ratings&page_size=10&query=hello",
+    {
+      fixture: "search-results.json",
+    }
+  );
+
+  cy.intercept(
+    "GET",
+    "https://freesound.org/apiv2/search/text/?fields=id%2Cname%2Cpreviews%2Cduration%2Cnum_downloads%2Cusername%2Cnum_ratings&page_size=10&page=2&query=hello",
+    {
+      fixture: "search-results-page-2.json",
+    }
+  );
```

Comment:
This intercept allows Cypress to mock the second page of search results for "hello" with the `search-results-page-2.json` fixture when clicking on the next page button while on the first page.